lib/galaxy_test/api/test_group_users.py

```diff
@@ -69,7 +69,7 @@ def test_update(self):
         self._assert_status_code_is_ok(update_response)
         group_user = update_response.json()
         self._assert_valid_group_user(group_user, assert_id=encoded_user_id)
-        assert group_user["url"] == f"/api/groups/{encoded_group_id}/users/{encoded_user_id}"
+        assert group_user["url"] == f"/api/groups/{encoded_group_id}/user/{encoded_user_id}"
```

Comment:
I wonder if the change is working with route alias, I guess both `user` and `users` should be valid for backward compatibility?

Reply:
Nevermind... this is correct :+1:

Reply:
Yep, I think we'll want to generate links for the non-aliased url.